block/src/Pos/Block/Logic/Header.hs

```diff
@@ -222,8 +223,11 @@ classifyHeaders inRecovery headers = do
     uselessGeneral =
         CHsUseless "Couldn't find lca -- maybe db state updated in the process"
     processClassify tipHeader = runMaybeT $ do
+        let hashes = getNewestFirst $ map (view headerHashG) headers
         lift $ logDebug $
-            sformat ("Classifying headers: "%listJson) $ map (view headerHashG) headers
+            sformat ("Classifying headers: "%buildListBounds)
```

Comment:
I think it'd be good to write `Classyfying headers (newest first):`, because newest first order may be a bit confusing (at least I usually expect oldest headers first).